test-prow-e2e.sh

```diff
@@ -13,6 +13,33 @@ function copyArtifacts {
   fi
 }
 
+function waitForAuthOperatorProgressing {
+  output=''
+  count=0
+  sleepDuration=15
+  maxRetries=16  #4 minutes
```

Comment:
If it usually takes 3 minutes, I'd make this more like 10 minutes.

Reply:
Considering @stlaz indicated "...that the operator should notice the config change in under 1 minute", I thought 4 mins was generous.